integration_test/third_party_apps_data/applications/jetty/metadata.yaml

```diff
@@ -43,12 +44,6 @@ expected_logs:
   - name: httpRequest.status
     type: string
     description: ''
-- log_name: syslog
```

Comment:
Removing this will prevent future releases reintroduce it back.

Temp fix: https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/pull/157